hoomd/md/AnisoPotentialPair.h

```diff
@@ -379,6 +442,33 @@ void AnisoPotentialPair< aniso_evaluator >::setParams(unsigned int typ1, unsigne
     h_params.data[m_typpair_idx(typ2, typ1)] = param;
     }
 
+template< class aniso_evaluator >
+void AnisoPotentialPair< aniso_evaluator >::setParamsPython(pybind11::tuple typ, pybind11::dict params)
```

Comment:
This isn't implemented in the child classes, and means we cannot actually set parameters in C++ from Python. Since the param_type of `AnisoPotentialPair` should always have a constructor from a Python `dict` we can just use that here.

Reply:
If that prevents testing from building which it probably will, then you will need to implement it for all child classes until all anisotropic pair potentials are converted.